lib/app/ctx.ex

```diff
@@ -624,4 +624,13 @@ defmodule App.Ctx do
     |> Repo.insert()
   end
 
+  @doc """
+  Create a basic session
+  """
+  def create_basic_session(user, attrs \\ %{}) do
+    Session.basic_changeset(user, attrs)
+    |> Repo.insert()
+  end
+
+
```

Comment:
@SimonLab this code looks good. 👍 
But do you feel that the `ctx.ex` file is getting a bit _large_ ...? 💭

Reply:
I think it's ok at the moment, most of the functions are only a couple of lines. The documentation is taking a bit more space but I don't think it's a bad thing. However I think we can remove some used functions (and add them back later on if necessary)